app/driver.go

```
@@ -199,6 +200,14 @@ func deployCSIDriver(kubeClient *clientset.Clientset, c *cli.Context, managerIma
 	namespace := os.Getenv(types.EnvPodNamespace)
 	serviceAccountName := os.Getenv(types.EnvServiceAccount)
 
+	rootDir, err := discoverKubeletRootDir(kubeClient, managerImage)
+	if err != nil {
+		rootDir = DefaultKubeletRootDir
+		logrus.Warnf("Failed to detect kubelet root dir path, using default root dir: %v", rootDir)
```

Expose error there. Almost never ignore the error when provided.